test.html

```diff
@@ -396,6 +396,24 @@ <h4>Coding LinkedIn Clone</h4>
           </div>
         </div>
       </div>
+
+      <!-- Event Viewer Section -->
+      <div class="event_viewer">
+        <input type="checkbox" id="toggle">
+        <label for="toggle" >
+          <div class="inputOption" id="event_button">
+            <i style="color: black" class="material-icons">add</i>
+            <div class="event_view_text">Event Viewer</div>
+          </div>
+        </label>
+        <div class="hidden" id="event_data">
+          <ul>"click" event occured at 11:04am</ul>
```

Comment:
little thing about the `ul` element, the `ul` or unordered list is the top level container for the list items or `li` elements, so how you want to use it is to instead do something like
```
<ul>
<li> item 1 </li>
<li> item 2 </li>
</ul>
```